TWLight/applications/views.py

```diff
@@ -960,13 +961,35 @@ def get_context_data(self, **kwargs):
         if app.parent and existing_authorization:
             context["previous_auth_expiry_date"] = existing_authorization.date_expires
 
-        # Get 5 recent applications
+        # Add recent applications in context
+        # get applications opened by editor in last 3 months
+        # if these applications are less than 5
+        # get some older applications to have atleast 5 recent apps
+        # else if these applications are more than or equal to 5
+        # simply add all of these to our context data
         # also exclude current app if it is present in recent apps
-        recent_apps = (
-            Application.objects.filter(editor=self.object.editor)
+
+        latest_recent_apps = (
+            Application.objects.filter(
+                editor=self.object.editor,
+                date_created__gte=datetime.today() - timedelta(days=90),
+            )
             .exclude(pk=app.pk)
-            .order_by("-id")[:5]
+            .order_by("-id")
         )
+        if len(latest_recent_apps) < 5:
```

Comment:
Per @Samwalton9's comment on this PR, we only need to show the applications made in the last 90 days, so we can remove this code (and the test you created for it).

Reply:
Yes :) removed this part and test for it.